bigquery_etl/glean_usage/templates/cross_channel.view.sql

```diff
@@ -0,0 +1,11 @@
+-- Generated via ./bqetl glean_usage generate
+CREATE OR REPLACE VIEW
+    `{{ project_id }}.{{ target_view }}`
+AS
+{% for (dataset, channel) in datasets -%}
+{% if not loop.first -%}
+UNION ALL
+{% endif -%}
+SELECT * REPLACE("{{ channel }}" AS normalized_channel)
+FROM {{ dataset }}.{{ table }}
```

Comment:
```suggestion
FROM `{{ project_id }}.{{ dataset }}.{{ table }}`
```

Following our convention of always using fully-qualified names in views.